scipy/optimize/_zeros_py.py

```diff
@@ -1415,11 +1414,11 @@ def _chandrupatla(f, x0, x1, *, args=(), xrtol=_xtol,
          with an arbitrary number of components of any type(s).
          `chandrupatla` seeks an array ``x`` (or scalar) such that each
          corresponding element in the output array (or scalar) are zero.
-    x0 : float, sequence, or ndarray
+    x1 : float, sequence, or ndarray
         The lower bound of the root of the function. Follows normal
         broadcasting rules and will be broadcast against `x1`and `f` before
         being passed into `f`.
-    x1 : float, sequence, or ndarray
+    x2 : float, sequence, or ndarray
         The upper bound of the root of the function. Follows normal
         broadcasting rules and will be broadcast against `x0` and `f` before
         being passed into `f`.
```

Comment:
Below:
`args` is additional arguments to be passed to `f`.
`success` and `status` are not attributes of `RootResult`; it's `converged` and `flag`, unfortunately.
Mention how `callback` can raise `StopIteration`